packages/blade-mcp/base-blade-template/package.json

```diff
@@ -10,7 +10,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@razorpay/blade": "^12.25.0",
+    "@razorpay/blade": "^12.28.0",
```

Comment:
can we use "latest" here? so it picks latest always? and probably we can remove that from our cli flag as well

Reply:
"latest" might end up fetching the latest version on consumer side as well after the app is built so if we ship breaking change, it might break all apps that have `latest` in their package.json 🤔

Reply:
that issue will also happen since we have added the `--latest` flag to new project creation cli as well? also this will only happen for people creating new projects so no breaking changes as such? since its new

Reply:
No no so when you run `npm install @razorpay/blade@latest` you install the latest version at that particular point in time when the command is running but your package.json would still have numbered version only `^12.31.0` for example.


But if you put "latest" in package.json itself, it means you're telling to install latest blade anytime the installation runs. So it can end up updating the version unexpectedly. 

package-lock.json does avoid unexpected updates but still the intent that you convey from package.json is to use latest blade version everytime you run `npm install`

Reply:
Separately running `npm install @razorpay/blade@latest` also ensures the latest version while scaffolding blade project. So it also goes and updates the latest version in package-lock.json

Vs if you just keep it in package.json, pacakge-lock will have a defined version which will keep on getting installed everytime unless you explicitly update package-lock in our templates

Reply:
Hmm that would help mostly in cases of preventing major version bumps right? like it won't allow going from `^12.31.0` to `13.0.0`?